examples/torch/requirements.txt

```diff
@@ -3,7 +3,7 @@ pillow>=8.0.1
 tensorboard>=2.1
 matplotlib>=3.3.3
 defusedxml>=0.7.0rc1
-mlflow==2.9.1
+mlflow>=2.9.2
```

Comment:
Better to use `~2.9.2` or `~2.9`, so that the patch or minor versions are up-to-date automatically, but at the same time no API-breaking major version changes happen without us knowing